reports/report_details.py

```diff
@@ -62,6 +62,14 @@ def add_transnational_to_regions(regions):
 WS_INFO = {
     'ws_01': {
         'name': '1',
+        '2015': {
+            'work_sheet': '1',
+            'historical': '1F',
+        },
+        '2010': {
+            'work_sheet': '1F',
+            'historical': None,
+        },
```

Comment:
My understanding is: `name` is the name of the currrent/new report sheet, and `historical` is the name of the old report sheet. With that, I expect the structure to be:
```python
    'ws_01': {
        '2015': {
            'name': '1',
            'historical': '1',
            .... other properties
        },
        '2010': {
            'name': '1',
            'historical': '1F',
            .... other properties
        },

```

1. 1F is historical in 2010 not 2015...
2. There is no need to try and optimize a single property. What if we want to change/tweak title for 2015 without changing how it was in 2010 since the reports are already out there?

Reply:
> 2. There is no need to try and optimize a single property. What if we want to change/tweak title for 2015 without changing how it was in 2010 since the reports are already out there?

@kilemensi I'm not sure I understand what you mean this. Can you kindly explain? Thanks

Reply:
It seems like you're trying to reduce changes or something @ascii-dev i.e. only making changes to properties/things that you think needs changing e.g. `name`/`historical` and you assume `tilte`, `description`, etc. will always be the same and hence they should be shared between 2010 and 2015... I'm saying there is no need for this optimization... just assume 2015 has it's own properties even if they're exact duplicates of 2010.

Reply:
Oh... I see what you mean here. So, I should move everything from the root keys and duplicate them within the `2015` and `2010` keys, updating only the values that are different. @kilemensi

Reply:
Yes @ascii-dev